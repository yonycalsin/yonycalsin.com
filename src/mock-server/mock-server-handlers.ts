import { rest } from 'msw'

import { API_ENDPOINTS } from 'services/shared'
import {
  getAllAchievementsApi,
  getAllBooksApi,
  getAllPagesApi,
  getAllProjectsApi,
  getCategoriesApi,
  getCategoryApi,
  getFeaturedAchievementsApi,
  getFeaturedRecommendationsApi,
  getPageApi,
  getPinnedProjectsApi,
  getPostApi,
  getPostsApi,
  getReadingBooksApi,
} from './mocks'

export const mockServerHandlers = [
  // Achievements
  rest.get(API_ENDPOINTS.ALL_ACHIEVEMENTS, getAllAchievementsApi),
  rest.get(API_ENDPOINTS.FEATURED_ACHIEVEMENTS, getFeaturedAchievementsApi),

  // Posts
  rest.get(API_ENDPOINTS.POST(':slug'), getPostApi),
  rest.get(API_ENDPOINTS.POSTS, getPostsApi),

  // Categories
  rest.get(API_ENDPOINTS.CATEGORY(':slug'), getCategoryApi),
  rest.get(API_ENDPOINTS.CATEGORIES, getCategoriesApi),

  // Books
  rest.get(API_ENDPOINTS.ALL_BOOKS, getAllBooksApi),
  rest.get(API_ENDPOINTS.READING_BOOKS, getReadingBooksApi),

  // Pages
  rest.get(API_ENDPOINTS.PAGE(':slug'), getPageApi),
  rest.get(API_ENDPOINTS.ALL_PAGES, getAllPagesApi),

  // Projects
  rest.get(API_ENDPOINTS.ALL_PROJECTS, getAllProjectsApi),
  rest.get(API_ENDPOINTS.PINNED_PROJECTS, getPinnedProjectsApi),

  // Recommendations
  rest.get(API_ENDPOINTS.FEATURED_RECOMMENDATIONS, getFeaturedRecommendationsApi),
]
