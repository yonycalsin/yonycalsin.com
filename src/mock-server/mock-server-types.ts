import type { DefaultBodyType, PathParams, ResponseComposition, RestContext, RestRequest } from 'msw'

export type MockRequest<Body = never, Params extends PathParams = PathParams<string>> = RestRequest<Body, Params>

export type MockResponse = ResponseComposition<DefaultBodyType>

export type MockContext = RestContext
