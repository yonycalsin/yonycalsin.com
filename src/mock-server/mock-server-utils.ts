function sanitizeEndpoint(endpoint: string) {
  const url = new URL(endpoint)

  if (!url.search) {
    return url.href
  }

  return url.href.replace(url.search, '')
}

export { sanitizeEndpoint }
