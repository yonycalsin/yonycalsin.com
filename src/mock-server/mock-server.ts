export async function initMocks() {
  if (typeof window === 'undefined') {
    const { server } = await import('./mock-server-node')

    server.listen()
  } else {
    const { worker } = await import('./mock-server-browser')

    worker.start()
  }
}
