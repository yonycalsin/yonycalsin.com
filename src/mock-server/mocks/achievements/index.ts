export * from './achievements-api'
export * from './achievements-mock'
