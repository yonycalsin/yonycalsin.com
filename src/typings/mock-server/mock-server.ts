import type { DefaultBodyType, PathParams, ResponseComposition, RestContext, RestRequest } from 'msw'

type MockRequest<Body extends DefaultBodyType = never, Params extends PathParams = PathParams<string>> = RestRequest<
  Body,
  Params
>

type MockResponse = ResponseComposition<DefaultBodyType>

type MockContext = RestContext

export type { MockContext, MockRequest, MockResponse }
