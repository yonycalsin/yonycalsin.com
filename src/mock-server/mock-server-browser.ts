import { setupWorker } from 'msw'

import { mockServerHandlers } from './mock-server-handlers'

export const worker = setupWorker(...mockServerHandlers)
