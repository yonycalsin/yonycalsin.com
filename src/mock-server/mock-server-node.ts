import { setupServer } from 'msw/node'

import { mockServerHandlers } from './mock-server-handlers'

function initServer() {
  const server = setupServer(...mockServerHandlers)

  return server
}

export default initServer
