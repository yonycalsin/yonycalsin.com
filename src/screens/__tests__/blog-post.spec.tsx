import { render, screen } from '@testing-library/react'

import type { BlogPostScreenProps } from 'typings/screens'
import { postsSuccess } from 'mock-server/mocks'
import { TestProvider } from 'tests'
import BlogPostScreen from 'screens/blog-post'

const MOCK_POST = postsSuccess.data[0]

const setup = (post: BlogPostScreenProps['post']) => {
  return render(
    <TestProvider>
      <BlogPostScreen post={post} />
    </TestProvider>,
  )
}

describe('BlogPost', () => {
  it('renders the blog post screen', () => {
    const view = setup(MOCK_POST)

    const title = screen.queryByRole('heading', { name: MOCK_POST.title })

    expect(title).toBeInTheDocument()

    const mdxContent = screen.queryByText(/hello from mdx content/i)

    expect(mdxContent).toBeInTheDocument()

    expect(view.baseElement).toMatchSnapshot()
  })
})
