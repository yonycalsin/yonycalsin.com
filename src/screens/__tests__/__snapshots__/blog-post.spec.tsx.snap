// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost renders the blog post screen 1`] = `
.emotion-0 {
  width: var(--yony-sizes-full);
  top: 0px;
  z-index: var(--yony-zIndices-banner);
  left: 0px;
  box-shadow: var(--yony-shadows-xs);
}

.chakra-ui-dark .emotion-0:not([data-theme]),
[data-theme=dark] .emotion-0:not([data-theme]),
.emotion-0[data-theme=dark] {
  box-shadow: var(--yony-shadows-lg);
}

.emotion-1 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding-top: var(--yony-space-5);
  padding-bottom: var(--yony-space-5);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media screen and (min-width: 48em) {
  .emotion-1 {
    padding-top: var(--yony-space-8);
    padding-bottom: var(--yony-space-8);
  }
}

.emotion-2 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
  display: contents;
}

.emotion-2:hover,
.emotion-2[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2:focus-visible,
.emotion-2[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-3>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: var(--yony-space-4);
  margin-inline-start: var(--yony-space-4);
}

.emotion-4 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: var(--yony-colors-primary-500);
  font-weight: var(--yony-fontWeights-bold);
}

.emotion-4:hover,
.emotion-4[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-4:focus-visible,
.emotion-4[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-6 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding-top: var(--yony-space-6);
  padding-bottom: var(--yony-space-12);
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-7>*:not(style)~*:not(style) {
  margin-top: var(--yony-space-2);
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

@media screen and (min-width: 48em) {
  .emotion-7>*:not(style)~*:not(style) {
    margin-top: var(--yony-space-5);
  }
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-10 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-11 {
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-fast);
  transition-timing-function: var(--yony-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
}

.emotion-11:hover,
.emotion-11[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-11:focus-visible,
.emotion-11[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-12 {
  -webkit-margin-start: 8px;
  margin-inline-start: 8px;
  -webkit-margin-end: 8px;
  margin-inline-end: 8px;
}

.emotion-15 {
  font-family: var(--yony-fonts-heading);
  font-weight: var(--yony-fontWeights-bold);
  font-size: var(--yony-fontSizes-3xl);
  line-height: 1.33;
}

@media screen and (min-width: 48em) {
  .emotion-15 {
    font-size: var(--yony-fontSizes-4xl);
    line-height: 1.2;
  }
}

.emotion-16 {
  color: gray;
  font-style: italic;
}

.emotion-17 {
  margin-top: 1.25rem;
  line-height: 1.7;
}

blockquote .emotion-17 {
  margin-top: 0px;
}

.emotion-18 {
  padding-top: var(--yony-space-12);
  padding-bottom: var(--yony-space-12);
}

.emotion-19 {
  width: 100%;
  -webkit-margin-start: auto;
  margin-inline-start: auto;
  -webkit-margin-end: auto;
  margin-inline-end: auto;
  max-width: var(--yony-sizes-container-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-20>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

.emotion-21 {
  font-weight: var(--yony-fontWeights-bold);
}

.emotion-22 {
  color: var(--yony-colors-gray-600);
}

.emotion-23 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: baseline;
  outline: 2px solid transparent;
  outline-offset: 2px;
  line-height: var(--yony-lineHeights-normal);
  border-radius: var(--yony-radii-md);
  font-weight: var(--yony-fontWeights-semibold);
  transition-property: var(--yony-transition-property-common);
  transition-duration: var(--yony-transition-duration-normal);
  height: auto;
  min-width: var(--yony-sizes-10);
  font-size: var(--yony-fontSizes-md);
  -webkit-padding-start: var(--yony-space-4);
  padding-inline-start: var(--yony-space-4);
  -webkit-padding-end: var(--yony-space-4);
  padding-inline-end: var(--yony-space-4);
  padding: 0px;
  color: var(--yony-colors-green-200);
}

.emotion-23:focus-visible,
.emotion-23[data-focus-visible] {
  box-shadow: var(--yony-shadows-outline);
}

.emotion-23:disabled,
.emotion-23[disabled],
.emotion-23[aria-disabled=true],
.emotion-23[data-disabled] {
  opacity: 0.4;
  cursor: not-allowed;
  box-shadow: var(--yony-shadows-none);
}

.emotion-23:hover,
.emotion-23[data-hover] {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-23:hover:disabled,
.emotion-23[data-hover]:disabled,
.emotion-23:hover[disabled],
.emotion-23[data-hover][disabled],
.emotion-23:hover[aria-disabled=true],
.emotion-23[data-hover][aria-disabled=true],
.emotion-23:hover[data-disabled],
.emotion-23[data-hover][data-disabled] {
  background: initial;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-23:active,
.emotion-23[data-active] {
  color: var(--yony-colors-green-500);
}

.emotion-24 {
  font-family: var(--yony-fonts-mono);
  background: whiteAlpha;
  border-radius: var(--yony-radii-md);
  border-width: 1px;
  border-bottom-width: 3px;
  font-size: 0.8em;
  font-weight: var(--yony-fontWeights-bold);
  line-height: var(--yony-lineHeights-normal);
  -webkit-padding-start: 0.4em;
  padding-inline-start: 0.4em;
  -webkit-padding-end: 0.4em;
  padding-inline-end: 0.4em;
  white-space: nowrap;
  -webkit-margin-start: var(--yony-space-2);
  margin-inline-start: var(--yony-space-2);
  -webkit-margin-end: var(--yony-space-2);
  margin-inline-end: var(--yony-space-2);
}

.emotion-26 {
  display: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: fixed;
  bottom: var(--yony-space-8);
  left: var(--yony-space-8);
}

.emotion-26>*:not(style)~*:not(style) {
  margin-top: var(--yony-space-5);
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

@media screen and (min-width: 62em) {
  .emotion-26 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

<body
  class="chakra-ui-dark"
>
  <div>
    <div
      class="emotion-0"
    >
      <div
        class="chakra-container emotion-1"
      >
        <a
          class="chakra-link emotion-2"
          href="/"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </span>
        </a>
        <div
          class="chakra-stack emotion-3"
        >
          <a
            class="chakra-link emotion-4"
            href="/projects"
          >
            Projects
          </a>
          <a
            class="chakra-link emotion-4"
            href="/me"
          >
            About Me
          </a>
        </div>
      </div>
    </div>
    <div>
      <div
        class="chakra-container emotion-6"
      >
        <header
          class="chakra-stack emotion-7"
        >
          <nav
            aria-label="breadcrumb"
            class="chakra-breadcrumb emotion-8"
          >
            <ol
              class="chakra-breadcrumb__list emotion-9"
            >
              <li
                class="chakra-breadcrumb__list-item emotion-10"
              >
                <a
                  class="chakra-breadcrumb__link emotion-11"
                  href="/"
                >
                  Home
                </a>
                <span
                  class="emotion-12"
                  role="presentation"
                >
                  <svg
                    color="gray.500"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 16 16"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M6.776 1.553a.5.5 0 0 1 .671.223l3 6a.5.5 0 0 1 0 .448l-3 6a.5.5 0 1 1-.894-.448L9.44 8 6.553 2.224a.5.5 0 0 1 .223-.671z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
              </li>
              <li
                class="chakra-breadcrumb__list-item emotion-10"
              >
                <a
                  class="chakra-breadcrumb__link emotion-11"
                  href="/blog"
                >
                  Blog
                </a>
              </li>
            </ol>
          </nav>
          <h2
            class="chakra-heading emotion-15"
          >
            How to mock a rest api
          </h2>
          <p
            class="chakra-text emotion-16"
          >
            Last updated at 
            14 Aug, 2022
          </p>
        </header>
        <div>
          <p
            class="emotion-17"
          >
            Hello from mdx content
          </p>
        </div>
      </div>
    </div>
    <footer
      class="emotion-18"
    >
      <div
        class="chakra-container emotion-19"
      >
        <div
          class="chakra-stack emotion-20"
        >
          <p
            class="chakra-text emotion-21"
          >
            Created By Yony Calsin
          </p>
          <p
            class="chakra-text emotion-22"
          >
            Built with Next.js, MDX, Chakra-Ui and Vercel
          </p>
          <button
            class="chakra-button emotion-23"
            type="button"
          >
            Press 
            <kbd
              class="chakra-kbd emotion-24"
            >
              Ctrl
            </kbd>
            +
            <kbd
              class="chakra-kbd emotion-24"
            >
              K
            </kbd>
             to start →
          </button>
        </div>
      </div>
    </footer>
    <div
      class="chakra-stack emotion-26"
    >
      <a
        class="chakra-link emotion-11"
        href="https://github.com/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
      <a
        class="chakra-link emotion-11"
        href="https://www.linkedin.com/in/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Linkedin
      </a>
      <a
        class="chakra-link emotion-11"
        href="https://twitter.com/yonycalsin"
        rel="noreferrer"
        target="_blank"
      >
        Twitter
      </a>
      <a
        class="chakra-link emotion-11"
        href="mailto:helloyonycalsin@gmail.com"
        rel="noreferrer"
        target="_blank"
      >
        Email
      </a>
    </div>
    <span
      hidden=""
      id="__chakra_env"
    />
  </div>
  <div
    class="chakra-portal"
  >
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top-left"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-top-right"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom-left"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
    />
    <ul
      aria-live="polite"
      id="chakra-toast-manager-bottom-right"
      role="region"
      style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
    />
  </div>
</body>
`;
